RELEASING.md

```diff
@@ -56,3 +53,12 @@ If review is not required, commit directly to `dev`.
    ```
 
 5. Announce new release in [Slack](https://kotlinlang.slack.com).
+
+# API reference documentation
```

Comment:
Could add a link to the [TeamCity build](https://buildserver.labs.intellij.net/buildConfiguration/Kotlin_KotlinSites_KotlinlangTeamcityDsl_KotlinxSerializationBuildApiReference) responsible for this, but it's the internal `buildserver` one, so not sure if it'll be of any use.

Could also add a link to the [configuration of the build trigger](https://github.com/JetBrains/kotlin-web-site/blob/fadd83777b02ea183dfd3bf668abb8fbb0653280/.teamcity/builds/apiReferences/vcsRoots/KotlinxSerialization.kt#L5), but again not sure if it'll be of any use.

Tell me if you think it's worth mentioning either one or both, I will.

Reply:
I don't mind sharing the link to the TeamCity build, it's up to you.